Abcrete/BangazonCLI
# Az cart

## Status
**READY**

## Description
Created 5th Menu Action method to add a product to the order 
## Todos
- [x ] Tests

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout az-cart
3. checkout to Bangazon folder
4. dotnet run
5. choose a customer
6. then choose option number 5
7. now you can choose product to add to the cart
8. check your db file it should have created new order column and new prodorder column

## Impacted Areas in Application
AddPoductToCartAction.cs
MenuManager.cs
OrderManager.cs
ProductManager.cs
Order.cs
Program.cs
## Link to Feature Ticket
#5
